### ISSUE TITLE
Version 0.8.20 of shims not in Maven

### ISSUE THREAD

**Issue by cheesinglee** [RoaringBitmap/RoaringBitmap#410]
**Describe the bug**
Attempting to update the RoaringBitmap dependency in our leiningen project to version 0.8.20, but I get an error saying that version 0.8.20 of the shims can not be found. Looking at some past issues here, I see that there have been problems pushing to Maven Central, but I thought I should bring this to attention in case it is an oversight.

**Java version:** openjdk 14.0.1

**Comment by lemire** [comment]
Thanks for the report. I do verify that it is not there:
https://repo1.maven.org/maven2/org/roaringbitmap/shims/

As I explained elsewhere, there is not much I can do. The files were synced. They have been uploaded as far as I can tell.

I am afraid that Maven Central is an obsolete and not-very-serious system. There is no trace, no transparency. It may as well be managed manually by a secretary behind some desk in some remote island.

I don't understand why the Java community tolerates something like that.

I'll remove the bug label. It may be a bug, but the bug is upstream. It is not with us. We did the release.
